libgeopmd/src/LevelZeroIOGroup.cpp

```diff
@@ -359,7 +359,7 @@ namespace geopm
                                                    domain_idx,
                                                    geopm::LevelZero::M_DOMAIN_ALL);
                                   },
-                                  1 / 1e6
+                                  1
```

Comment:
For this and the other time & timestamp signals:

Should we also update the description text to remove the "in seconds" notes since the L0 docs don't commit to specific units?

Reply:
https://github.com/geopm/geopm/pull/3615/commits/2a9b54d97ce1770d1c02448dd9dc770f9cba7902